README.md

```diff
@@ -47,12 +47,12 @@ Elements are described in the position paper
-| [DeepLabCut](https://github.com/datajoint/element-deeplabcut)                                         | [Workflow for pose tracking](https://github.com/datajoint/workflow-deeplabcut)                                                                                                       | [Background, features, and status](docs/description/deeplabcut.md)             |
-| [Event- and trial-based experiments](https://github.com/datajoint/element-event)                      | [Workflow for Neuropixels](https://github.com/datajoint/workflow-array-ephys) <br/> [Workflow for multiphoton laser scanning](https://github.com/datajoint/workflow-calcium-imaging) | [Background, features, and status](docs/description/event.md)                  |
-| [Electrode localization for Neuropixels](https://github.com/datajoint/element-electrode-localization) | [Workflow for Neuropixels](https://github.com/datajoint/workflow-array-ephys)                                                                                                        | [Background, features, and status](docs/description/electrode_localization.md) |
-| [Facemap](https://github.com/datajoint/element-facemap)                                               | In development                                                                                                                                                                       | [Background, features, and status](docs/description/facemap.md)                |
+| [DeepLabCut](https://github.com/datajoint/element-deeplabcut)                                         | [Workflow for pose tracking](https://github.com/datajoint/workflow-deeplabcut)                                                                                                       | [Link](docs/description/deeplabcut.md)             |
+| [Event- and trial-based experiments](https://github.com/datajoint/element-event)                      | [Workflow for Neuropixels](https://github.com/datajoint/workflow-array-ephys) <br/> [Workflow for multiphoton laser scanning](https://github.com/datajoint/workflow-calcium-imaging) | [Link](docs/description/event.md)                  |
+| [Electrode localization for Neuropixels](https://github.com/datajoint/element-electrode-localization) | [Workflow for Neuropixels](https://github.com/datajoint/workflow-array-ephys)                                                                                                        | [Link](docs/description/electrode_localization.md) |
+| [Facemap](https://github.com/datajoint/element-facemap)                                               | [Workflow for Facemap](https://github.com/datajoint/workflow-facemap)                                                                                                                                                                       | [Link](docs/description/facemap.md)                |
```

Comment:
Adding a table row for visual stimulation. The spacing here looks laid out to vertically match in an IDE. I recommend installing an IDE package that automatically folds links in markdown (e.g., [sublime example](https://github.com/SublimeText-Markdown/MarkdownEditing)) to make editing easier in the future
```suggestion
| [Facemap](https://github.com/datajoint/element-facemap)                                               | [Workflow for Facemap](https://github.com/datajoint/workflow-facemap)                                                                                                                | [Link](docs/description/facemap.md)                |
| [Visual Stimulation](https://github.com/datajoint/element-visual-stimulus/)                           | [Workflow for Visual Stimulation](https://github.com/datajoint/workflow-visual-stimulus/)                                                                                            | [Link](docs/description/visual_stimulus.md)        |
```

Reply:
Thanks!  I will look into this package.